overlay.d/05rhcos/usr/lib/dracut/modules.d/41rhcos-check-luks-syntax/rhcos-fail-boot-for-older-luks-syntax.service

```diff
@@ -0,0 +1,16 @@
+# Fixes the encryption issue with LUKS in 4.7 when a user 
+# tries to use the RHCOS 4.6 or earlier LUKS syntax in an 
+# Ignition config.
+[Unit]
+Description=Fail boot if Ignition uses an older syntax for enabling LUKS in 4.7 
+Before=ignition-disks.service
+After=ignition-fetch.service
+ConditionKernelCommandLine=ignition.firstboot
+
+[Service]
+Type=oneshot
+ExecStart=/usr/libexec/rhcos-fail-boot-for-older-luks-syntax
+RemainAfterExit=yes
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
We shouldn't need this since it's statically enabled via `ignition-complete.target`.